### PR TITLE
Update dependencies and GitHub actions

### PR DESCRIPTION
**Pull request by CoachYT1**

### CODE REVIEW

**Comment by derklaro** [comment]
Do you mind updating mockito and adventure as well? (maybe there are other dependency updates which I didn't see) 🤔

File: pom.xml [hunk 1]
```diff
@@ -17,8 +17,8 @@
 		<project.fullVersion>${project.version}</project.fullVersion>
 
 		<junit.version>5.8.2</junit.version>
-		<mockito.version>4.3.1</mockito.version>
-		<netty.version>4.1.74.Final</netty.version>
+		<mockito.version>4.4.0</mockito.version>
+		<netty.version>4.1.75.Final</netty.version>
```
Comment (dmulloy2) [inline pom.xml:21]: fwiw, the netty version spigot uses is [4.1.68](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/pom.xml#106) (so close)

Comment (CoachYT1) [reply pom.xml:21]: NachoSpigot is planning to bump Netty to 4.1.75

Comment (derklaro) [reply pom.xml:21]: It makes no difference here as that netty version is only used in tests (minecraft doesn't bundle netty anymore) 🤔

**Pull request closed without merging**